CHANGELOG.next.asciidoc

```diff
@@ -79,8 +79,6 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 
 - Add _bucket to histogram metrics in Prometheus Collector {pull}11578[11578]
 - Prevent the docker/memory metricset from processing invalid events before container start {pull}11676[11676]
-- Stop overwriting cloud.* fields from add_cloud_metadata if they are not empty. {pull}11612[11612] {issue}11305[11305]
```

Comment:
I cannot find the other instance of this line?

Reply:
@exekias This was the earliest changelog entry. Then with recommendations, I worded to the new one instead 😄